scripts/lint-translations.sh

```diff
@@ -1,25 +1,13 @@
-# extract new strings
-yarn i18n:extract
-
 # get all translation files that were updated
 # from `yarn i18n:extract`
-LOCALE_CHANGED_FILES=$(git diff --name-only src/locales)
+CHANGED_FILES=$(git diff --name-only src/locales/**/*.po)
```

Comment:
PRs shouldn't contain any changes to `.po` files anymore. Should only contain changes to `.pot` file.

We'll run this script in the CI for PRs :)